tools/buildtools/ambertools/parmchk2.xml

```diff
@@ -0,0 +1,81 @@
+<tool id="parmchk2" name="ParmChk2" version="@VERSION@">
```

Comment:
Oh one last thing. Should we prefix all tool ids again with `amber_`?

Reply:
ambertools_

Reply:
committing in a mo